app/services/main_conversation/tools/websocket_utils/handle_websocket_connection.py

```
@@ -104,7 +105,7 @@ async def handle_websocket_connection(websocket: WebSocket):
                 except WebSocketDisconnect:
                     logger.info("WebSocket connection closed while sending error")
                     break
-    except WebSocketDisconnect:
+    except WebSocketDisconnect as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)